app/views/games/index.html.erb

```diff
@@ -4,7 +4,8 @@
 <% @games.each do |game| %>
   <% if game.open_seat? %>
```

Comment:
That branch was pushed to master? There is no open_seats method available. Maybe you accidentally pulled it in the merge request? And if thats a method that just returns true or false, its good practice to have a (?) on the end like so:  .open_seats?  It just makes it visually easier to see.